dataset-snippets/README.md

```diff
@@ -31,8 +31,28 @@ The instructions are identical for both the SQL snippets and the Dataset Builder
 ## Snippet tests
 To test individual snippets such as plots, the best thing to do is copy and paste them into a notebook on the workbench.
 
-See also https://github.com/all-of-us/workbench-snippets/issues/30
+## Integration 'smoke tests'
+The script to auto-generate the Jupyter Snippets Menu configuration also emits both `r_dataset_snippets_menu_config_smoke_test.R` and `py_dataset_snippets_menu_config_smoke_test.py`. Those scripts each include, respectively, all the R Dataset Builder snippets and all the Python Dataset Builder snippets. If those scripts are run from the workbench environment and there are no obvious bugs in the snippets, they will run start-to-finish without error and emit _"Smoke test complete!"_ when they have completed successfully. (This won't necessarily catch all bugs, but its a good start.)
+
+The Dataset from Dataset Builder is defined in [r_dataset_snippets_menu_config.smoke_test_setup](./r_dataset_snippets_menu_config.smoke_test_setup) and [py_dataset_snippets_menu_config.smoke_test_setup](./py_dataset_snippets_menu_config.smoke_test_setup), respectively. Update it as needed.
+
+To run the R Dataset Builder snippets smoke tests:
+```
+%%bash
+
+Rscript r_dataset_snippets_menu_config_smoke_test.R  # There will be output, but there should be no errors.
```

Comment:
Do these not also use WORKSPACE_CDR?

Reply:
Not at the moment.
* What Dataset Builder emits has the CDR hard-coded. I pasted that into these `*.smoke_test_setup` files.
* I wanted to minimize local edits, so the only difference between [r_dataset_snippets_menu_config.smoke_test_setup](https://github.com/deflaux/workbench-snippets/blob/add-dataset-to-smoke-test/build/r_dataset_snippets_menu_config.smoke_test_setup#L25) and the code that would be exported to an R notebook by Dataset Builder is just `s/dataset_123456789/YOUR-DATASET-NAME/g`

Please see the comment at the top of [that file]((https://github.com/deflaux/workbench-snippets/blob/add-dataset-to-smoke-test/build/r_dataset_snippets_menu_config.smoke_test_setup#L25)) and let me know if you recommend something different.

Reply:
Thanks you're right, I got these mixed up - I thought I was commenting on the non-datasets documentation.